x-pack/plugins/security_solution/common/endpoint/data_generators/endpoint_rule_alert_generator.ts

```diff
@@ -279,6 +279,15 @@ export class EndpointRuleAlertGenerator extends BaseDataGenerator {
               value: '',
             },
           ],
+          response_actions: [
```

Comment:
I think this means that every alert generated will have an automated response on it, right? should that be randomized? so that some have it but other don't?

Reply:
Yes, you're right - this applies to every alert. However, it doesn't make alert to have the response on, because in order to show responses we have to trigger actions with alert_id, but it means that the `rule` that triggered the alert had the automated response action configured. Only reason to have this in config in test is that the `Results` tab in Event details flyout requires rule to have response_actions in parameters - otherwise we do not show it. So I would say it's safe to have it in all alerts, as long as we do not want to hide it on purpose ;p